src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -1148,6 +1148,13 @@ RefCountedPtr<LoadBalancingPolicy::Config> ChooseLbPolicy(
   } else {
     policy_name = grpc_channel_args_find_string(resolver_result.args,
                                                 GRPC_ARG_LB_POLICY_NAME);
+    if (!LoadBalancingPolicyRegistry::LoadBalancingPolicyExists(policy_name,
+                                                                nullptr)) {
+      gpr_log(GPR_ERROR, "grpc.lb_policy_name: %s passed through "
```

Comment:
Please combine these two log messages into one, and make it a bit more readable:

```
gpr_log(GPR_ERROR, "LB policy \"%s\" does not exist; using pick_first instead",
        policy_name);
```

Reply:
Fixed in the latest commit.